pom.xml

```diff
@@ -101,16 +126,45 @@
                 <configuration>
                     <argLine>${surefireArgLine}</argLine>
                     <skipTests>${skip.unit.tests}</skipTests>
-                    <excludes>
-                        <exclude>src/main/webapp</exclude>
-                        <exclude>src/test/java/com/epam/test_generator/dao</exclude>
-                        <exclude>src/test/java/com/epam/test_generator/dto</exclude>
-                        <exclude>src/test/java/com/epam/test_generator/file_generator</exclude>
-                        <exclude>src/test/java/com/epam/test_generator/services</exclude>
-                        <exclude>src/test/java/com/epam/test_generator</exclude>
-                    </excludes>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <version>2.20.1</version>
+                <executions>
+                    <execution>
+                        <id>failsafe-integration-tests</id>
+                        <phase>integration-test</phase>
+                        <goals>
+                            <goal>integration-test</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <argLine>${failsafeArgLine}</argLine>
+                    <skipTests>${skip.integration.tests}</skipTests>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
```

Comment:
What is that plugin for?

Reply:
This plugin is needed to add more than one test sources